### DISCUSSION TITLE
sinatra support ruby 3?

### DISCUSSION THREAD

**Discussion by ghost** [sinatra/sinatra#1730] [Q&A]
if not are some plans to support it in the near future?

**Comment by svoop** [comment]
Check this out: https://github.com/sinatra/sinatra/actions/runs/1657870243

**Comment by dentarg** [comment]
Yeah master branch supports Ruby 3 (and 3.1 it looks like). The latest release (v2.1.0) almost supports Ruby 3.0 too, I noted the exceptions that I know of at https://github.com/sinatra/sinatra/pull/1733#issuecomment-1004146942

**Reply by svoop** [reply]
Time to cut a release then, after all, it's the season for it. :wink:

**Reply by dentarg** [reply]
The people who can doesn't seem to be around anymore :/

**Reply by svoop** [reply]
@zzak @namusyaka Happy New Year to you! Do you see a chance to [cut a release](https://github.com/sinatra/sinatra/pull/1733#issuecomment-1003853939) these days? There seems to be some momentum and it would be a waste to pass it, Sinatra is too cool to slumber into irrellevance. 

In case you need a hand, would you consider adding more owners to help speeding up future rollouts?
